README.md

```diff
@@ -76,8 +76,10 @@ render(<Picker />, mountNode);
 | getPopupContainer | function(trigger) |  | to set the container of the floating layer, while the default is to create a div element in body |
 | onChange | Function(date: moment, dateString: string) |  | a callback function, can be executed when the selected time is changing |
 | onOpenChange | Function(open:boolean) |  | called when open/close picker |
-| onFocus | (evnet:React.FocusEventHandler<HTMLInputElement>) => void |  | called like input's on focus |
-| onBlur | (evnet:React.FocusEventHandler<HTMLInputElement>) => void |  | called like input's on blur |
+| onFocus | (event:React.FocusEventHandler<HTMLInputElement>) => void |  | called like input's on focus |
+| onBlur | (event:React.FocusEventHandler<HTMLInputElement>) => void |  | called like input's on blur |
+| onKeyDown | (event:React.KeyboardEventHandler<HTMLInputElemet>) => void |  | input on keydown event |
+| disableKey | Array<String> |  | disable default key for keydown event, available only for key 'ENTER', 'TAB', & 'ESC' |
```

Comment:
`onKeyDown` is enough, I think we should not add built-in `disableKey` feature in Picker.

Reply:
okay, i will remove ```disableKey```

currently the ```onKeyDown``` will run alongside the default key, it is necessary to override the default key? or just leave it to run with ```onKeyDown``` props?

Reply:
we can disable default behaviors of `onKeyDown`  when it is passed by props.

Reply:
did you mean ```onKeyDown``` will have 2 parameters: ```e``` & ```defaultKeyDownBehaviors```, where the ```defaultKeyDownBehaviors``` is function that can be called?

Reply:
or did you mean we need to add custom ```preventDefault``` to ```event```, that will prevent the default behaviors?

Reply:
`defaultKeyDownBehaviors` that i think is origin behaviors of `rc-picker` defined.

Reply:
the sample code:
``` js
const originKeyDown = (e: React.KeyboardEvent<HTMLInputElement>) => {
    switch (e.which) {
      case KeyCode.ENTER: {
        if (!open) {
          triggerOpen(true);
        } else if (onSubmit() !== false) {
          setTyping(true);
        }

        e.preventDefault();
        return;
      }

      case KeyCode.TAB: {
        if (typing && open && !e.shiftKey) {
          setTyping(false);
          e.preventDefault();
        } else if (!typing && open) {
          if (!forwardKeyDown(e) && e.shiftKey) {
            setTyping(true);
            e.preventDefault();
          }
        }
        return;
      }

      case KeyCode.ESC: {
        setTyping(true);
        onCancel();
        return;
      }
    }

    if (!open && ![KeyCode.SHIFT].includes(e.which)) {
      triggerOpen(true);
    } else if (!typing) {
      // Let popup panel handle keyboard
      forwardKeyDown(e);
    }
  }

  const inputProps: React.DOMAttributes<HTMLInputElement> = {
    onMouseDown: () => {
      setTyping(true);
      triggerOpen(true);
    },
    onKeyDown: e => {
      if (onKeyDown) {
        onKeyDown(e, originKeyDown.bind(null, e))
        return
      }
      originKeyDown(e)
    }
    // ...
  }
```

Reply:
Sorry just read your comment.

So if using `onKeyDown`, the default behaviors is not doing its work unless the user call it, right?

Reply:
Yes, user don't need to pass `onKeyDown`, if user wants to use default behaviors.

But there is a premise, this is a new feature that has never been seen before, otherwise it will be a break change.

@afc163  What do you think?

Reply:
You can view in my last commit, i use arguments that can be called if user want to prevent default behavior

I use that as parameters, so user can call `preventDefaultBehaviors()` like they call `event.preventDefault()` as usual

CMIIW

Reply:
Both solutions are ok.

Waiting thoughts of @afc163

Reply:
can i ask? 
why the code coverage check is gone?

i just update this branch from master